src/mapml-viewer.js

```diff
@@ -324,6 +324,16 @@ export class MapViewer extends HTMLElement {
         totalSize += 49;
         this._fullScreenControl = M.fullscreenButton().addTo(this._map);
       }
+      if (this.controlslist.toLowerCase().includes("geolocation") && !this._geolocationButton && (totalSize + 49) <= mapSize){
+        totalSize += 49;
+        this._geolocationButton = M.geolocationButton({
+          showPopup: false,
```

Comment:
I was listening to the page using NVDA, maybe we should change the strings [option](https://github.com/domoritz/leaflet-locatecontrol#possible-options), so it says "locate button" or something of that nature.